core/src/main/java/org/jboss/logmanager/LoggerNode.java

```diff
@@ -136,8 +136,17 @@ public void reap(Reference<Logger, LoggerNode> reference) {
     LoggerNode(final LogContext context) {
         parent = null;
         fullName = "";
-        handlersUpdater.clear(this);
```

Comment:
This needs to stay or we need to initialize the array with an empty array to avoid an NPE in the `LoggerNode.addHandler()`.

Reply:
Ah, I misread the original code.  Fixing.